Tools/Parser/UnicodeData.cpp

```diff
@@ -61,6 +61,8 @@ UnicodeData::UnicodeData(const string &directory) :
     istringstream dataStream(m_data, ios::binary);
     uint32_t codePoint;
 
+    bool inRange = false;
+    uint32_t rangeFirstCodePoint;
```

Comment:
Assign some initial value to this variable.

Reply:
Done.